src/components/Searchbar/InstantSearch.tsx

```diff
@@ -21,37 +27,62 @@ export const InstantSearch = () => {
     setSearch(e.target.value);
 
     await axios
-        .get(`https://jsonplaceholder.typicode.com/albums`)
-        .then((res) => {
-          setPosts(res.data);
-        })
-        .catch((error) => {
-          if (axios.isCancel(error) || error) {
-            console.log('Could not get');
-          }
-        });
+      .get(`https://jsonplaceholder.typicode.com/albums`)
+      .then((res) => {
+        setPosts(res.data);
+      })
+      .catch((error) => {
+        if (axios.isCancel(error) || error) {
+          console.log('Could not get');
+        }
+      });
   };
 
   return (
     <div className="searchModule">
-      <div className="container">
-      <input
-        value={search}
-        onChange={(e) => onLsChange(e)}
-        type="text"
-        placeholder="Почніть вводити назву..."
-      />
-
-      <ul>
-        {searchRes.map((res) => {
-          return <li key={res.id}>{res.title}</li>;
-        })}
-      </ul>
+      <div className="searchModule__input ">
+        <input
+          value={search}
+          onChange={(e) => onLsChange(e)}
+          type="text"
+          placeholder="Почніть вводити назву..."
+        />
+
+        <ul>
+          {searchRes.map((res) => {
+            return <li key={res.id}>{res.title}</li>;
+          })}
+        </ul>
+      </div>
+
+      <div className="searchModule__select">
+        <Select
+          label="Виберіть епізод"
+          items={ANIME_GENRES}
+          selectedKey={genr}
+          placeholder="Виберіть жанр"
+          bgNone
+          maxWidth={250}
+          isLoading={isGenreLoading || !ANIME_GENRES}
+          onSelectionChange={(selected) => {
+            setGenr(selected);
+          }}
+        >
+          {(item) => (
+            <Item textValue={item.title} id={item.id}>
+              {item.title}
+            </Item>
+          )}
+        </Select>
       </div>
 
-      <div className="container">
-        <button className="button-style" type="button">Пошук</button>
-      </div>      
+      <button
+        style={{ height: '54px' }}
```

Comment:
check if this height is required

Reply:
if you remove it, then: 
![image](https://github.com/GGO-web/Unimay/assets/97807120/6685ae13-d8bf-4d86-9662-b14ab6ee3052)

Reply:
made position absolute for block UL 
![image](https://github.com/GGO-web/Unimay/assets/97807120/f6f4fd85-52bf-425b-97f2-0ba82573c783)